packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts

```diff
@@ -161,19 +159,25 @@ const init = async () => {
     ])
     .as('securityEventUids');
 
+  const accountCustomerUids = AccountCustomers.query()
+    .distinct('uid')
```

Comment:
Are there non-distinct uids?

Reply:
No, I just got copy-paste happy.  Thank you for catching that.